analyzer/src/funTest/assets/projects/synthetic/gradle-expected-output-scopes-excludes.yml

```diff
@@ -0,0 +1,185 @@
+---
```

Comment:
The commit message still talks about "skipExclusions" instead of "skipExcludes".

Reply:
Similar also for follow-up commits.

Reply:
I hope I found all stale references.